index.html

```diff
@@ -195,6 +195,33 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ S A G A R Contributor card END ________  -->
 
+        <!-- ________ Subhayu Dutta Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Subhayu Dutta</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/subhayudutta0" target="_blank">Your handle</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/subhayudutta" target="_blank">Your handle</a>
```

Comment:
Hi, thank you for your contribution, please note that you should also update "Your handle" with your username.
You can commit and push again, then we'll merge your PR, thanks

```suggestion
            <a href="https://twitter.com/subhayudutta0" target="_blank">@subhayudutta0</a>
            <i class="fab fa-github"></i>
            <a href="https://github.com/subhayudutta" target="_blank">Also replace this one</a>
```